CI.hs

```diff
@@ -101,13 +103,28 @@ genVersionStr = \case
 
 parseOptions :: Opts.Parser Options
 parseOptions = Options
-    <$> Opts.switch
-        ( Opts.long "upload-to-bintray"
-       <> Opts.help "If specified, will try uploading the sdists to Bintray, using credentials in BINTRAY_BASIC_AUTH env var.")
-    <*> Opts.option readFlavor
+    <$> Opts.option readFlavor
         ( Opts.long "ghc-flavor"
        <> Opts.help "The ghc-flavor to test against"
         )
+    <*> Opts.strOption
+        ( Opts.long "da-base-sha"
+       <> Opts.help "DA flavour only. Base commit to use from the GHC repo."
+       <> Opts.showDefault
+       <> Opts.value "ghc-8.8.1-release"
```

Comment:
Did you just create this tag? If not, I don’t understand what this is pointing to.

Reply:
Not my addition, we were already checking out that tag. As far as I can tell it's [the official release tag](https://gitlab.haskell.org/ghc/ghc/-/tree/ghc-8.8.1-release) on the GHC repo.

Reply:
Makes sense, I got a bit confused and thought this was something we created.